webrender/examples/blob.rs

```diff
@@ -11,14 +11,12 @@ extern crate webrender;
 mod boilerplate;
 
 use boilerplate::{Example, HandyDandyRectBuilder};
-use rayon::Configuration as ThreadPoolConfig;
-use rayon::ThreadPool;
+use rayon::{Configuration as ThreadPoolConfig, ThreadPool};
```

Comment:
we really need to ensure that rustfmt is in agreement with servo tidy here, could you verify?

Reply:
I ran rustfmt nightly on blob.rs and it make some changes. Those changes still ran cleanly through servo-tidy, so I think it's just that servo-tidy is more stringent than rustfmt currently.